server/memphis_handlers_schemas.go

```diff
@@ -685,3 +715,40 @@ func (sh SchemasHandler) RollBackVersion(c *gin.Context) {
 	c.IndentedJSON(200, extedndedSchemaDetails)
 
 }
+
+func (sh SchemasHandler) ValidateSchema(c *gin.Context) {
+	var body models.ValidateSchema
+	ok := utils.Validate(c, &body, false, nil)
+	if !ok {
+		return
+	}
+
+	isValid := false
+	var response gin.H
+	schemaType := strings.ToLower(body.SchemaType)
+	err := validateSchemaType(schemaType)
+
+	if err != nil {
+		serv.Warnf(err.Error())
+		c.AbortWithStatusJSON(configuration.SHOWABLE_ERROR_STATUS_CODE, gin.H{"message": err.Error()})
+		return
+	}
+
+	schemaContent := body.SchemaContent
+	err = validateSchemaContent(schemaContent, schemaType)
+
+	if err != nil {
+		serv.Warnf(err.Error())
+		response = gin.H{
+			"is_valid": isValid,
+			"error":    err.Error(),
+		}
+
+		c.AbortWithStatusJSON(SCHEMA_VALIDATION_ERROR_STATUS_CODE, response)
```

Comment:
my mistake in this case return a regular error response containing the validation error